### PR TITLE
Update the command documentation and add types

### PR DESCRIPTION
**Pull request by j-f1**

### CODE REVIEW

File: commands.md [hunk 1]
```diff
@@ -155,23 +155,59 @@ For an example, here's the implementation of the `ArrayExpression` node type:
...
+}
+
+declare function concat(docs: Doc<>[]): Doc<"concat">
+declare function indent(docs: Doc<>[]): Doc<"indent">
+declare function align(n: number, docs: Doc<>[]): Doc<"align">
+declare function group(docs: Doc<>[], opts?: {
+                        shouldBreak?: boolean,
+                        expandedStates?: Doc<>[]
+                      }): Doc<"group">
+declare function conditionalGroup(alternatives: Doc<>[], opts?: {
```
Comment (duailibe) [inline commands.md]: `ident`, `align` and `group` receive only a doc, not array of docs

File: commands.md [hunk 2]
```diff
@@ -155,23 +155,59 @@ For an example, here's the implementation of the `ArrayExpression` node type:
...
           path.map(print, "elements")
         )
       ])
-    ),
+    ]),
     line,
     "]"
   ])
-)
+])
```
Comment (duailibe) [inline commands.md]: Per the other comment, this code was correct before

**Comment by azz** [comment]
I worry that these types will become drift out of sync.

**Comment by j-f1** [comment]
@azz I’ve moved the types to be inline with the docs. If anyone makes a breaking change, I’d assume they would update the docs too. Right now, it’s unclear what exactly each function takes as parameters.

**Comment by azz** [comment]
Thanks for the docs!

**Pull request merged**